pkg/daemon/ceph/agent/flexvolume/manager/ceph/manager.go

```diff
@@ -52,17 +52,23 @@ type pathFinder interface {
 }
 
 // NewVolumeManager create attacher for ceph volumes
-func NewVolumeManager(context *clusterd.Context) *VolumeManager {
+func NewVolumeManager(context *clusterd.Context) (*VolumeManager, error) {
 	vm := &VolumeManager{
 		context:          context,
 		devicePathFinder: &devicePathFinder{},
 	}
-	vm.Init()
-	return vm
+	err := vm.Init()
+	return vm, err
 }
 
 // Init the ceph volume manager
 func (vm *VolumeManager) Init() error {
+	// check if the rbd is a builtin kernel module, if it is then we don't need to load it manually
+	in, err := sys.IsBuiltinKernelModule(rbdKernelModuleName, vm.context.Executor)
```

Comment:
Can you check the error here before the usage of `in`

Reply:
Sure.